docs/components/alert/index.md

```diff
@@ -8,13 +8,17 @@
 
 Alert is an inline component used for displaying different types of messages.
 
+<components-status react='released' vue='released' elements='released' />
+
 ## Example
 
 <theme-switcher />
 
 <alert-example></alert-example>
 
 <tabs-content> 
+  <template #usage disabled>
```

Comment:
Should this `disabled` prop be here? 🤔

Reply:
...and should we really have that "Usage" tab there? 🤔 A lot of components doesn't have it? And it really feels like the odd bird amongst the other framework implementations? 😄 

![image](https://github.com/warp-ds/tech-docs/assets/7531505/a906b15d-eded-46ac-bed8-de9ecbb97646)
![image](https://github.com/warp-ds/tech-docs/assets/7531505/d8efdd3d-967b-4180-83f4-46f3e6a7000c)

Reply:
haha the image 😂 Okay cool, let's go by @BalbinaK's suggestion instead then and add it as part of the index page instead 🙌